### ISSUE TITLE
[BUG] Unable to connect to any of the services

### ISSUE THREAD

**Issue by danzay369** [rivafarabi/deckboard#883]
Hi, i faced the problem, that i cannot connect any of the services, neither twitch, spotify, or twitter. When i press the "login" button it just opens the white blank window
[log.log](https://github.com/rivafarabi/deckboard/files/9276754/log.log)

**Comment by danzay369** [comment]
okay, i reinstalled the app, and it updated, and the problem is fixed

**Comment by danzay369** [comment]
well, it happened again, i guess i will leave this issue open, if it keeps reappearing

**Comment by danzay369** [comment]
[log.log](https://github.com/rivafarabi/deckboard/files/9315618/log.log)
i tried all of them, but what i'm planning to use is only twitch connection
sorry for long reply

**Comment by rivafarabi** [comment]
Do you have any software like an anti virus that possibly can intercept the
connection between the app and the network?

**Comment by danzay369** [comment]
I have a Kaspersky. I tried to connect with antivirus offline, doesn't help.

**Comment by rivafarabi** [comment]
Sorry for the late reply, need to investigate this issue further. 

I conducted an experiment by disabling several antiviruses, and as the result, Kaspersky is one of the antiviruses that intercept the connection between the server app with the internet.

I also released version [2.0.5-rc1](https://github.com/rivafarabi/deckboard/releases/tag/v2.0.5-rc1), which includes edited user agent in the auth browser window. You might want to try this release if disabling Kaspersky is not working.
